dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java

```diff
@@ -35,10 +35,10 @@ public static boolean isDevMode () {
 	}
 
 	public static String getDynamicContentPath() {
-		String realPath = Config.getStringProperty("DYNAMIC_CONTENT_PATH");
-		if (!UtilMethods.isSet(realPath)) {
-			realPath = com.liferay.util.FileUtil.getRealPath("/dotsecure");
-		}
+		String realPath = Try.of(()-> com.liferay.util.FileUtil.getRealPath(
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'com.liferay.util.FileUtil.getRealPath' due to existing import 'com.liferay.util.FileUtil'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5816746)